app/src/components/Modals/activity-modal/activity-form-modal.tsx

```diff
@@ -85,6 +84,15 @@ const AddActivityModal: FC<AddActivityModalProps> = ({
     methodology: methodology,
   });
 
+  const { emissionsFactorTypes, emissionsFactorsLoading } = useEmissionFactors({
```

Comment:
Calling it `areEmissionsFactorsLoading` might make it more clear that it's a boolean.